### PR TITLE
Hotfix Italian.xml

### LINKED ISSUE #12265: Italian mistranslation: "Alternate icones"
### Description of the Issue

Italian mistranslation
Settings-->Preferences-->general-->"Alternate icones" is mistranslated "Alterna Icone "; the correct translation is "**icone alternative**"

### Debug Information
Notepad++ v8.4.6   (64-bit)
Build time : Sep 25 2022 - 19:51:39
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Notepad++\change.log" 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 22H2
OS Build : 22621.521
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    DSpellCheck (1.4.24)
    mimeTools (2.8)
    NppConverter (4.4)
    NppExport (0.4)
    NPPJSONViewer (1.40)
    NppXmlTreeviewPlugin (2)
    XMLTools (3.1.1.13)

immagine

### PR DESCRIPTION
**Pull request by conky77**
fix #12265 and improve translation

### CODE REVIEW

File: PowerEditor/installer/nativeLang/italian.xml [hunk 1]
```diff
@@ -878,23 +878,23 @@
...
+                    <Item id="6101" name="Barra degli strumenti"/>
                     <Item id="6102" name="Nascondi"/>
-                    <Item id="6103" name="Fluent UI: Icone piccole"/>
-                    <Item id="6104" name="Fluent UI: Icone grandi"/>
-                    <Item id="6129" name="Filled Fluent UI: Icone piccole"/>
-                    <Item id="6130" name="Filled Fluent UI: Icone grandi"/>
+                    <Item id="6103" name="Icone piccole"/>
+                    <Item id="6104" name="Icone grandi"/>
+                    <Item id="6129" name="Icone piene (piccole)"/>
+                    <Item id="6130" name="Icone piene (grandi)"/>
```
Comment (rdipardo) [inline PowerEditor/installer/nativeLang/italian.xml:886]: Ma non è questo un po' oscuro? Voglio dire, a proprio che cosa si oppone la "pienezza" che manca alle *altre* icone piccole?

Forse così?

- Icone __*di contorno*__ (piccole)
- Icone __*di contorno*__ (grandi)
- Icone piene . . . ecc.

Comment (conky77) [reply PowerEditor/installer/nativeLang/italian.xml:886]: @rdipardo grazie del suggerimento, ho fatto come hai suggerito (avevo messo così sbirciando la spagnola). grazie ciao

**Comment by donho** [comment]
@conky77 
I don't understand - how come a lot of entries were not translated correctly on the first time?

**Comment by conky77** [comment]
> @conky77 I don't understand - how come a lot of entries were not translated correctly the first time?

I just tried to make the translation uniform using the same terms for all entries and make the Italian more natural. To do this, I had also a look at other Latin translations and other software and debug the translation I don't have any problem if you want to involve other people or the traditional translator. It would be easier to translate if we had a tool to translate than to manage an XML. When we translate, we don't have time to debug the translation, and sometimes we don't know what we are translating. It would help if he knew in advance when the release candidate will be released. If you wanted to involve Ices-eyes, for me it's ok.

**Comment by conky77** [comment]
@rdipardo you reviewed this pull request. Could you please review also the last version? 
Hai già revisionato la versione precedente. Potresti per cortesia revisionare o confermare l'ultima versione? Grazie

File: PowerEditor/installer/nativeLang/italian.xml [hunk 2]
```diff
@@ -1160,7 +1160,19 @@ Puoi indicare serie di delimitatori di colonna usando degli spazi per separare i
 (non selezionarlo se non sai cosa stai facendo)"/>
                 </Delimiter>
 
-                <Cloud title="Cloud e link">
+                <Performance title="Prestazioni">
+                    <Item id="7141" name="Restrizioni per File di grandi dimensioni"/>
+                    <Item id="7143" name="Abilita restrizioni per i file di grandi dimensioni (no evidenziazione sintattica"/>
```
Comment (rdipardo) [inline PowerEditor/installer/nativeLang/italian.xml:1165]: (no evidenziazione sintattica_

Bisogna la parentesi chiusa

Comment (conky77) [reply PowerEditor/installer/nativeLang/italian.xml]: @rdipardo, thanks, done. Do you agree with the others modifications? Are they better than the previous translation?

**Comment by donho** [comment]
@conky77 
OK, thank you for your explanation.
I suppose it's the last time there's the important change in Italian localization.
If it happens again, then there's a problem.

File: PowerEditor/installer/nativeLang/italian.xml (hunk 2 above)
Comment (rdipardo) [reply PowerEditor/installer/nativeLang/italian.xml]: > @rdipardo, thanks, done. Do you agree with the others modifications? Are they better than the previous translation?

Tutto bene per quanto ci vedo. Buon lavoro!

**Pull request closed without merging**